docs/api.md

```diff
@@ -0,0 +1,344 @@
+
+**Request Body Required?** Yes  
+**Request Body Format** JSON  
+**Request Attributes**
+
+| Name | Description                                    | Type   | Required? |
+| ---- | ---------------------------------------------- | ------ | --------- |
+| kid  | The ID of the key used to encrypt the data     | String | Yes       |
+| enc  | The encryption algorithm                       | String | Yes       |
+| cty  | Content type of the key. Should always be JWK. | String | Yes       |
+| data | Encrypted credential info                      | String | Yes       |
+
+**Request Example**
+
+```JSON
+{
+  "kid": "867fghjkl",
+  "enc": "A256GCM",
+  "cty": "b5+jwk+json",
+  "data": "rdfthyukjlA4nmajhgf"
+}
+```
+
+### Response to POST /api/v1/accounts/:account_id/items
+
+**Response Body Format** JSON
+
+**Response Statuses**
+
+| Outcome | Status Code        | Notes                                               |
+| ------- | ------------------ | --------------------------------------------------- |
+| Success | 201 Created        |                                                     |
+| Failure | 400 Bad Request    | Missing required attributes or has extra attributes |
+| Success | 401 Unauthorized   | No auth token or bad auth token                     |
+| Success | 406 Not Acceptable | "Accept" header does not include JSON               |
+
+**Response Examples**
+
+_Status: 201 Created_
+
+```JSON
+{
+  "id": 8910,
+  "self": "http://localhost:8080/accounts/1234/items/8910"
+}
+```
+
+_Status: 400 Bad Request_
+
+```JSON
+{
+  "Error": "The request is missing one or more required attributes"
+}
+```
+
+## View all Credential Overviews for an Account
+
+GET /api/v1/accounts/:account_id/items
+
+### Request to GET /api/v1/accounts/:account_id/items
+
+**Path Parameters**
+
+| Name       | Description           |
+| ---------- | --------------------- |
+| account_id | The user's account ID |
+
+**Request Body Required?** No  
+**Request Body Format** N/A  
+**Request Attributes** N/A  
+**Request Example** N/A
+
+### Response to GET /api/v1/accounts/:account_id/items
+
+**Response Body Format** JSON  
+**Response Statuses**
+
+| Outcome | Status Code        | Notes                                 |
+| ------- | ------------------ | ------------------------------------- |
+| Success | 200 OK             |                                       |
+| Success | 401 Unauthorized   | No auth token or bad auth token       |
+| Success | 406 Not Acceptable | "Accept" header does not include JSON |
+
+**Response Examples**
+
+```JSON
+[
+  {
+    "id": 8910,
+    "kid": "867fghjkl",
+    "enc": "A256GCM",
+    "cty": "b5+jwk+json",
+    "data": "rdfthyukjlA4nmajhgf",
+    "self": "http://localhost:8080/api/v1/accounts/1234/items/8910"
+  },
+  {
+    "id": 5678,
+    "kid": "867fghjkl",
+    "enc": "A256GCM",
+    "cty": "b5+jwk+json",
+    "data": "jemccouandsfbfrh7au",
+    "self": "http://localhost:8080/api/v1/accounts/1234/items/2345"
```

Comment:
"self": "http://localhost:8080/api/v1/accounts/1234/items/2345"
should the item id be 5678?